notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # if the whole dataset cannot fit into GPU memory (or even CPU RAM -- use mmap for that).
</pre><p>It would help to either elaborate upon the <code>mmap</code> comment or remove it. I thin I understand what you are saying but I'm not expecting it to be immediately obvious to a user.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>